vNext/src/Orleans/Orleans.csproj

```diff
@@ -33,6 +33,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.0.0-preview2-final" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.0.0-preview2-final" />
```

Comment:
Yes this PR is not mergeable yet. There's some TODOs(see description) which need to be addressed before merging. I'd like to do some of there todos after the two sln merged into one sln.